bundles/org.openhab.binding.kodi/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -560,4 +562,16 @@
 		<description>Profile of Kodi</description>
 		<state pattern="%s"/>
 	</channel-type>
+	<channel-type id="screensaver">
```

Comment:
Please also update the default translation properties

Reply:
Fixed